packages/labs/src/labs.md

```diff
@@ -228,6 +228,8 @@ Modifiers are the tool through which you customize Popper.js's behavior. It defi
 
 @## Tooltip 2.0
 
+`Tooltip2` uses [`Popover2`](#labs.popover-2.0) under the hood. The public API is the same as [`Tooltip`](#core/components/tooltip), though note that the `isDisabled` prop has been renamed to `disabled`. Read
```

Comment:
1. Can you kick text to a few different lines to achieve shorter line length in the markdown file?
2. There's a trailing word, `Read`, that seems superfluous.